relay-monitors/src/lib.rs

```diff
@@ -59,8 +59,7 @@ struct CheckIn {
     check_in_id: Uuid,
 
     /// Identifier of the monitor for this check-in.
-    #[serde(serialize_with = "uuid_simple")]
-    monitor_id: Uuid,
+    monitor_slug: String,
```

Comment:
Do we need to consider backward compatibility?

Reply:
No, the SDK has not been released and we can break the API for crons currently.